code/_onclick/ai.dm

```diff
@@ -42,15 +44,23 @@
 	var/turf/pixel_turf = get_turf_pixel(A)
 	if(isnull(pixel_turf))
 		return
-
-	if(!can_see(A) && isturf(A))
+	if(!can_see(A))
+		if(isturf(A)) //On unmodified clients clicking the static overlay clicks the turf underneath
+			return //So there's no point messaging admins
+		message_admins("[ADMIN_LOOKUPFLW(src)] might be running a modified client! (failed can_see on AI click of [A] (Turf Loc: [ADMIN_VERBOSEJMP(pixel_turf)]))")
+		var/message = "[key_name(src)] might be running a modified client! (failed can_see on AI click of [A] (Turf Loc: [AREACOORD(pixel_turf)]))"
```

Comment:
There is currently a bug on both tg and tgmc that spuriously allows ais to see the whole map so this isnt 100% accurate, should elaborate that that might also be the case

Reply:
I dont think there is a need to make the string longer, most of the time when this happens admins either ask each other or ping me about it.

Reply:
Okey dokey then